### DISCUSSION TITLE
System Error Encountered while trying to enable vibrancy

### DISCUSSION THREAD

**Discussion by YimingXie0702** [illixion/vscode-vibrancy-continued#25] [Q&A]
Hi there, I am a big fan of this extension and I'm having an issue installing this on my new laptop. The error is 
`Command 'Enable Vibrancy' resulted in an error (A system error occurred (EROFS: read-only file system, mkdir '/private/var/folders/mx/4zf1th9s4wlg09fm091vm16w0000gn/T/AppTranslocation/7D31DE4F-8D48-4DE1-A873-2B8253AF24EE/d/Visual Studio Code.app/Contents/Resources/app/out/vscode-vibrancy-runtime-v6'))`
My laptop is MacOS with M1 chip, vibrancy continued version 1.1.5, and VS Code version is 1.71.2 (Universal)/

Thank you!

**Comment by illixion** [comment]
You need to move Visual Studio Code.app to your /Applications folder, otherwise macOS applies a security mitigation that blocks the extension from working.

**Comment by KiraH4cknet** [comment]
I already have the vscode in /applications folder, but I still get the error, even run the application with administrator permissions, will it have something to do with this in the beta of MacOS Ventura?

**Reply by illixion** [reply]
Could you please try something and see if that fixes it? Apparently this issue can happen due to the quarantine flag not being removed in certain scenarios. Either use Finder to move the VSCode out of `/Applications` and then back in (or move it into `/Applications` if it wasn't there already), or run the following terminal command:

```shell
xattr -dr com.apple.quarantine "/Applications/Visual Studio Code.app"
```
